actor/v7action/application_summary.go

```diff
@@ -206,6 +213,21 @@ func (actor Actor) addDroplet(summary ApplicationSummary) (DetailedApplicationSu
 	}, allWarnings, nil
 }
 
+func (actor Actor) addDeployment(detailedSummary DetailedApplicationSummary) (DetailedApplicationSummary, Warnings, error) {
+	var allWarnings Warnings
+
+	deployment, warnings, err := actor.GetLatestActiveDeploymentForApp(detailedSummary.GUID)
+	allWarnings = append(allWarnings, warnings...)
+	if err != nil {
+		if _, ok := err.(actionerror.ActiveDeploymentNotFoundError); !ok {
```

Comment:
**nit/non-blocking** Could we use errors package for this instead? It's more modern go and more readable : E.g. 

```
if err != nil && !errors.Is(err, actionerror.ActiveDeploymentNotFoundError{}) {
	return DetailedApplicationSummary{}, allWarnings, err
}
```

Reply:
Fixed